tests/docker/Dockerfile

```diff
@@ -64,6 +64,16 @@ RUN mkdir -p "/opt/kafka-2.3.1" && chmod a+rw /opt/kafka-2.3.1 && curl -s "$KAFK
     make install && \
     ldconfig
 
+# install dependencies for kafka clients such as sarama
+RUN go get github.com/Shopify/sarama && \
+    cd /root/go/src/github.com/Shopify/sarama/examples/interceptors && go mod tidy && go build && \
+    cd /root/go/src/github.com/Shopify/sarama/examples/http_server && go mod tidy && go build && \
+    cd /root/go/src/github.com/Shopify/sarama/examples/consumergroup && go mod tidy && go build && \
+    cd /root/go/src/github.com/Shopify/sarama/examples/sasl_scram_client && go mod tidy && go build
+
+# Expose port 8080 for any http examples within clients
+EXPOSE 8080
```

Comment:
@ivotron any issue here with parallel tests?

Reply:
this should be fine. The `EXPOSE` directive is actually a no-op one, meaning it doesn't really cause the port to be published when running the container; it's only there for documentation purposes. When running this image, users still need to map the ports.